tests/test_cmdline.py

```diff
@@ -916,12 +916,9 @@ def test_cmdline_on_ignored_undeclared_option(
                 output_format = 'human_detailed'
                 # code = ['.']
                 deps = ['foobar']
-                # pyenv = None
-                # custom_mapping = None
                 # ignore_undeclared = []
                 # ignore_unused = []
-                # deps_parser_choice = None
-                # install_deps = False
```

Comment:
As stated above, we should consider _not_ removing the `None`-valued  directives here, as that makes them less visible to the user as potential customization points.

Either that, or we should consider a bigger change where we omit default-valued directives altogether. (The argument being that the user should rather look to our documentation for potential customization points. Not sure I agree with that, though...)

Reply:
Tests fixed to replace `None` with the `...` placeholder in revised [cd44db9](https://github.com/tweag/FawltyDeps/pull/300/commits/cd44db9ed259b358338a61341975c8990a2df9f6).